server/log/main.go

```diff
@@ -55,6 +64,27 @@ func checkDatabaseAccessible(dbUri string) error {
 	return err
 }
 
+func loadPrivateKey(keyFile, keyPassword string) (crypto.KeyManager, error) {
+	if len(keyFile) == 0 || len(keyPassword) == 0 {
```

Comment:
They can load it with pem.Decode() or do we really want to allow that case here?